checkstyle/checkstyle-suppressions.xml

```diff
@@ -28,5 +28,8 @@
     <suppress checks="EmptyLineSeparator|GenericWhitespace|SingleSpaceSeparator|NoWhitespaceBefore|WhitespaceAround|org.apache.ignite.tools.checkstyle.OverrideAnnotationOnTheSameLineCheck"
               files="modules[\/\\]h2[\/\\].*\.java"/>
 
+    <suppress checks="EmptyLineSeparator|GenericWhitespace|SingleSpaceSeparator|NoWhitespaceBefore|WhitespaceAround|org.apache.ignite.tools.checkstyle.OverrideAnnotationOnTheSameLineCheck"
+              files="modules[\/\\]indexing[\/\\]src[\/\\]main|test[\/\\]java[\/\\]org[\/\\]apache[\/\\]ignite[\/\\]internal[\/\\]processors[\/\\]query[\/\\]stat[\/\\]hll[\/\\].*\.java"/>
```

Comment:
```suggestion
              files="modules[\/\\]indexing[\/\\].*[\/\\]hll[\/\\].*\.java"/>
```
Also if change the pattern like this^, the checkstyle plugin will start to fail with violations (all of them seem legit)